src/runtime.cpp

```diff
@@ -276,7 +290,8 @@ runtime::next_outcome runtime::get_next()
 
     if (resp.has_header(DEADLINE_MS_HEADER)) {
         auto const& deadline_string = resp.get_header(DEADLINE_MS_HEADER);
-        unsigned long ms = strtoul(deadline_string.c_str(), nullptr, 10);
+        constexpr int decimal_radix = 10;
```

Comment:
This one feels stupid. I don't love the magic numbers check (disabled it in all of the CRT libs), but this one just feels extra unnecessary.

Reply:
Sigh. Yes. You're absolutely right. I hate it.
P.S I rewrote this comment 3 times starting with: "I'm feeling worried about future people working on the project" to "I'm close to nuking it if it complains one more time" to "You're right, screw this"

Reply:
![](https://i.imgflip.com/3i7aqt.jpg)

Reply:
Decided to keep the magic number warning

    I was about to remove the constexpr variable and replace the argument
    with the hardcoded value with an inline comment and I realized that this
    is exactly what the warning/error is about. Documentation/readability.